examples/s2_field_combination_extern_rf_train/nrw_crop_extern_s2_workflow_to_datasets_with_timeshift.ipynb

```diff
@@ -0,0 +1,2364 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using the bounding box, a time interval and a resolution, we define the area of interest as a temporal raster space-time cube."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'spatialBounds': {'upperLeftCoordinate': {'x': 421395, 'y': 5727833},\n",
+       "  'lowerRightCoordinate': {'x': 476201, 'y': 5681078}},\n",
+       " 'timeInterval': {'start': '2021-01-01T00:00:00.000+00:00',\n",
+       "  'end': '2021-12-31T00:00:00.000+00:00'},\n",
+       " 'spatialResolution': {'x': 10.0, 'y': 10.0}}"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from datetime import datetime\n",
+    "time_start = datetime(2021, 1, 1)\n",
+    "time_end = datetime(2021, 12, 31)\n",
+    "\n",
+    "study_area = ge.api.RasterQueryRectangle(\n",
+    "    spatialBounds=ge.SpatialPartition2D(xmin, ymin, xmax, ymax).to_api_dict(),\n",
+    "    timeInterval=ge.TimeInterval(time_start, time_end).to_api_dict(),\n",
+    "    spatialResolution=ge.SpatialResolution(10.0, 10.0).to_api_dict(),\n",
+    ")\n",
+    "study_area"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For each band (B02, B03, B04, B08) as well as the scene mask (SCL), we create a workflow, that downloads the Sentinel-2 data for the area of interest and stores it as a new dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{}"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "status_download = {}\n",
+    "\n",
+    "for b in [\"B02\", \"B03\", \"B04\", \"B08\", \"SCL\"]:\n",
+    "    print(b)\n",
+    "    sentinel2_band_workflow = ge.unstable.workflow_blueprints.sentinel2_band(b)\n",
+    "    sentinel2_band_workflow_id = ge.register_workflow(sentinel2_band_workflow.to_workflow_dict())\n",
+    "    sentinel2_band_workflow_dataset_task = sentinel2_band_workflow_id.save_as_dataset(study_area, f\"sentinel2_nrw_crop_10m_{b}\")\n",
+    "    sentinel2_band_workflow_dataset_task.wait_for_finish(print_status=False)\n",
+    "    print(sentinel2_band_workflow_dataset_task.get_status())\n",
+    "    status_download[b] = sentinel2_band_workflow_dataset_task.get_status()\n",
+    "\n",
+    "status_download"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Check the quota used for downloading the bands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0"
```

Comment:
This shows that it does not work :smile:

Reply:
it does. i just think it is a waste of time to rerun this all the time when i have the data already